core/shared/src/test/scala/cats/parse/ParserTest.scala

```diff
@@ -1341,17 +1378,17 @@ class ParserTest extends munit.ScalaCheckSuite {
       val pa: Parser[Any] = p1.fa
       val pb: Parser[Any] = p2.fa
       assertEquals(pa.orElse(pb), pa | pb)
-    }
+    } &&
```

Comment:
these tests were not running. It is kind of okay since they were mostly compilation tests, but still...

we were discarding non-unit values. Not okay.

Reply:
Not to throw shade at scalac, but the C# compiler would have given me a warning here.

Okay, that's totally to throw shade at scalac.